node-examples/neopixel-i2c/main.js

```diff
@@ -0,0 +1,22 @@
+const { requestI2CAccess } = require("node-web-i2c");
+const NEOPIXEL_I2C = require("@chirimen/neopixel-i2c");
+const { promisify } = require("util");
+const sleep = promisify(setTimeout);
+
+main();
+
+async function main() {
+  const i2cAccess = await requestI2CAccess();
+  const port = i2cAccess.ports.get(1);
+  const neopixel_i2c = new NEOPIXEL_I2C(port, 0x41);
```

Comment:
こちらもlowerCamelCaseかつi2cは自明なので省略したほうがすっきりと見えます
```suggestion
  const neopixel = new Neopixel(port, 0x41);
```